.gitignore

```diff
@@ -8,3 +8,4 @@ captures/
 /data/music/wip*.pomf
 /data/music/WIP*.pomf
 /webroot/fvudata/*
+pio_*.ini
```

Comment:
Maybe put them into a folder e.g. `/local/`?

Reply:
for user includes it does not matter since it's ignored in `.gitignore`, or there could any other files in `/local`?

Reply:
Not sure, but having loose local files aside from .env in the repo root feels wrong. That way at least we can gitignore the whole folder and if more local files pop up later then they can just go in there.